### ISSUE TITLE
AltGr-e resolves to e on Swedish keymap - should be €

### ISSUE THREAD

**Issue by Ben3eeE** [atom/atom-keymap#174]
![altgre](https://cloud.githubusercontent.com/assets/1058982/20225587/70f38712-a844-11e6-825b-964195eba6de.gif)

Workaround:
```
atom.keymaps.addKeystrokeResolver(({event}) => {
  if (event.code === 'KeyE' && event.altKey && event.ctrlKey) {
    return '€'
  }
})
```

**Comment by Ben3eeE** [comment]
@nathansobo This happens for every non-latin key and might be why we saw reports of AltGraph keys not working at all since the update. If I use Russian layout <kbd>Ctrl-Alt-o</kbd> resolves to `o` making it impossible to use the current keybinding for add project folder.

**Comment by nathansobo** [comment]
Closed via #181
